tests/lib/nsxv_test.py

```diff
@@ -861,15 +956,16 @@ def testNsxvStrForDestinationIpv4Mismatch(self):
     # parse the xml
     root = ET.fromstring(sections[1])
 
-    # check destination has only Ipv6
-    destination_address = root.findall('./rule/destinations/destination')
-    self.assertIsNotNone(destination_address)
-    for dest_addr in destination_address:
-      addr_type = dest_addr.find('type').text
-      self.assertEquals(addr_type, 'Ipv6Address')
+    source_addr = root.findall('./rule/sources/source')
+    dest_addr = root.findall('./rule/destinations/destination')
 
-    self.naming.GetNetAddr.assert_has_calls(
-        [mock.call('SOME_HOST'), mock.call('GOOGLE_DNS')])
+    return source_addr, dest_addr
+
+  def verifyAddressType(self, address, type):
```

Comment:
Not clear about this change. How will making type a list help?  Can you please let me know?
Or else this method should take the source addr, destination add, type and loop through each source addr and destination addr and check the type.